Dino runner: new svg's + responsivity + bacteria spawning

vickyad opened the pull request.

vickyad committed:
Feat: svg's changed + background more responsive

vickyad committed:
Fix: bacteria spawning fixed

LeoHernandes reviewed (commented):
Aqui funciononu direitinho as bacterias spawnando e os svg estão perfeitinhos, uma delicia

Só sugiro tirar então a variável que eu fiz "isFirstJump" q dava um tempo maior no início do jogo para a bacteria vir, pq agr ela já está num tempo bom em todo o jogo e demora bastante para vir na primeira vez. Não sei se pra ti está assim também.

prphawk reviewed (commented):
só pra mim que parece que ficou bem mais difícil? por conta do tempo que ele fica no ar e o pouco tempo de reação com a bactéria no mobile

vickyad committed:
Update: isFirstJump removed

vickyad committed:
Merge branch 'main' into dino-runner-design

LeoHernandes committed:
fix( route to empty_dino.svg & arrowback color in dino slider)

prphawk committed:
Feat(image on the customize button on dinogotchi)

LeoHernandes committed:
fix (dinoRunner fixes)

- dino size
- bacteria speed
- no gameover dialog
- bacteria apearing as intended
- score displaying correctly

LeoHernandes committed:
Merge remote-tracking branch 'origin/dino-runner-design' into dino-runner-design
